SimPEG/Regularization.py

```diff
@@ -745,16 +746,16 @@ def _update_regmesh_indActive(self, change):
         if getattr(self, 'regmesh', None) is not None:
             self.regmesh.indActive = change['value']
 
-    @properties.validator('cell_weights')
-    def _validate_cell_weights(self, change):
-        if change['value'] is not None:
-            # todo: residual size? we need to know the expected end shape
-            if self._nC_residual != '*':
-                assert len(change['value']) == self._nC_residual, (
-                    'cell_weights must be length {} not {}'.format(
-                        self._nC_residual, len(change['value'])
-                    )
-                )
+    # @properties.validator('cell_weights')
```

Comment:
why was this commented out?

Reply:
No clue. I will uncomment and rerun tests